### PR TITLE
add Flink validation endpoints support

### PR DESCRIPTION
**Pull request by Serpentiel**
- adds Flink validation endpoints support

### CODE REVIEW

File: flink_job.go [hunk 1]
```diff
@@ -88,6 +88,24 @@ type (
...
+	}
+
+	// ValidateFlinkJobResponse Aiven API response
+	// POST https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/job/validate
+	ValidateFlinkJobResponse struct {
+		APIResponse
+
+		JobValidateError struct {
+			Message  string        `json:"message"`
+			Position flinkPosition `json:"position"`
```
Comment (ivan-savciuc) [inline flink_job.go:106]: where `Position` is gonna be used?

**Review by ivan-savciuc — APPROVED** [review]
LGTM

File: flink.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package aiven
+
+type (
+	// shared fields by some responses
+	flinkPosition struct {
```
[the remark this answers is not in the document]
Comment (Serpentiel) [reply flink.go:5]: we are supposed to provide full Aiven API support, according to the `README.md`

File: flink_job.go (hunk 1 above)
Comment (Serpentiel) [reply flink_job.go:106]: wherever end-users need it :)

**Pull request merged**